x-pack/metricbeat/main.go

```diff
@@ -2,13 +2,6 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
-/*
```

Comment:
Remove on purpose? Cleanup?

Reply:
This is a bugfix that I couldn't manage to fix yet. In some moment one of the scripts removes this comment from the package name, I guess that to replace it with the license version. AFAIK there is not easy / simple way to avoid it than to rewrite the headers script.

Reply:
Let's keep it like this then.

@andrewkroh might have an idea ...